### ISSUE TITLE
Could you share the hyperparameters and training logs for Efficientnet?

### ISSUE THREAD

**Issue by Mingyuam** [tensorflow/models#8669]
Hi,
I'm trying to reproduce the performance for Efficientnet-b0 with multi-GPUs. Could you please share the hyperparameters and training logs for efficientnet? It will be very helpful to have the loss-curve to compare with my current run. Thank you!

**Comment by saberkun** [comment]
Hyperparameters are here: https://github.com/tensorflow/models/tree/master/official/vision/image_classification/configs/examples/efficientnet/imagenet
I don't think we are allowed to share loss-curve or logs unless we strip a lot of internal side logs.
We do plan to release tensorboard curves through tensorboard.org in the future.

**Comment by Mingyuam** [comment]
Thank you for your reply! I understand, but could you please let me know how many epochs you train for each model to converge to the promised performance? Did you train all of the models for 500 epochs?
I appreciate your help!

**Comment by ameyparanjape** [comment]
@Mingyuam Did you train this on Imagenet data or custom data?

**Comment by Mingyuam** [comment]
> @Mingyuam Did you train this on Imagenet data or custom data?

Imagenet
